apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -12,226 +23,95 @@ const InvertedWrapper: React.FC = ({ children }) => {
-  })
-  .addStory('Primary with Label Above', () => <Spinner labelPosition="above" label="Loading" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Label Below', () => <Spinner labelPosition="below" label="Loading" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Extra Tiny', () => <Spinner size="extra-tiny" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Tiny', () => <Spinner size="tiny" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Extra Small', () => <Spinner size="extra-small" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Small', () => <Spinner size="small" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Medium', () => <Spinner size="medium" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Large', () => <Spinner size="large" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Size Extra Large', () => <Spinner size="extra-large" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('Primary with Huge', () => <Spinner size="huge" />, {
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
+  ))
+  .addStory('Primary', () => <Spinner className={useStyles().paused} />, {
     includeHighContrast: true,
     includeDarkMode: true,
   })
   .addStory(
-    'Inverted',
+    'Primary with Label',
     () => (
-      <InvertedWrapper>
-        <Spinner appearance="inverted" />
-      </InvertedWrapper>
+      <div className={useStyles().paused} style={{ display: 'flex', flexDirection: 'column', rowGap: '20px' }}>
+        <Spinner labelPosition="before" label="Loading" />
+        <Spinner labelPosition="after" label="Loading" />
+        <Spinner labelPosition="above" label="Loading" />
+        <Spinner labelPosition="below" label="Loading" />
+      </div>
     ),
     {
       includeHighContrast: true,
       includeDarkMode: true,
+      includeRtl: true,
     },
   )
   .addStory(
-    'Inverted with Label',
+    'Primary + size',
     () => (
-      <InvertedWrapper>
-        <Spinner appearance="inverted" label="Loading" />
-      </InvertedWrapper>
+      <div className={useStyles().paused} style={{ display: 'flex', flexDirection: 'column', rowGap: '20px' }}>
+        <Spinner size="extra-tiny" />
+        <Spinner size="tiny" />
+        <Spinner size="extra-small" />
+        <Spinner size="small" />
+        <Spinner size="medium" />
+        <Spinner size="large" />
+        <Spinner size="extra-large" />
+        <Spinner size="huge" />
```

Comment:
Maybe add labels here as well? Wouldn't hurt to test the combination of size + label.